### ISSUE TITLE
Fix interface configuration for Wildfly 9

### ISSUE THREAD

**Issue by therek** [coi-gov-pl/puppet-jboss#14]
Interface configuration atrributes `any-ipv4-address` and `any-ipv6-address` have been removed in Wildfly 9. The interface configuration in `puppet-jboss` should be fixed to take that into account and retain backward compatibility.

The error message:
```
Warning: JBoss CLI command failed, try 1/3, last status: 1, message: {
    "outcome" => "failed",
    "result" => {},
    "failure-description" => {"host-failure-descriptions" => {"wbcdev" => "WFLYCTL0201: Unknown attribute 'any-ipv4-address'"}},
    "rolled-back" => true
}
```

Available interface configuration attributes:
```
{
    "outcome" => "success",
    "result" => {
        "any" => undefined,
        "any-address" => true,
        "inet-address" => undefined,
        "link-local-address" => undefined,
        "loopback" => undefined,
        "loopback-address" => undefined,
        "multicast" => undefined,
        "name" => "public",
        "nic" => undefined,
        "nic-match" => undefined,
        "not" => undefined,
        "point-to-point" => undefined,
        "public-address" => undefined,
        "site-local-address" => undefined,
        "subnet-match" => undefined,
        "up" => undefined,
        "virtual" => undefined
    }
}
```

**Comment by therek** [comment]
Fixed in v1.0.4.

**Comment by cardil** [comment]
You meant it is fixed in PR #29 and is planned for v1.0.4 release.

**Comment by therek** [comment]
That's precisely what I meant :)
